doc/reference/guidelines.rst

```diff
@@ -6,24 +6,31 @@
 Odoo Guidelines
 ===============
 
-This page introduce the new Odoo Coding Guidelines. These guidelines
-aim to improve the quality of the code (better readability of source,
-...) and  Odoo Apps. Indeed, proper code ought ease maintenance, aid
-debugging, lower complexity and promote reliability.
+This page introduce the new Odoo Coding Guidelines. These guidelines aim to improve the quality of the code (better readability of source, ...) and Odoo Apps. Indeed, proper code ought ease maintenance, aid debugging, lower complexity and promote reliability.
```

Comment:
- introduces
- Those aim to improve ... (remove redundant guidelines)
- remove parenthesis, use i.e. or e.g., never know which one to use
- proper code eases maintenance, aids debugging, lowers complexity and promotes reliability